modules/javafx.graphics/src/main/java/com/sun/glass/utils/NativeLibLoader.java

```diff
@@ -169,7 +170,7 @@ private static void loadLibraryInternal(String libraryName, List<String> depende
                 //is recognized by existence of JNI_OnLoad_libraryname() C function.
                 //If libraryname contains hyphen, it needs to be translated
                 //to underscore to form valid C function indentifier.
-                if ("iOS".equals(System.getProperty("os.name"))
+                if ("ios".equals(System.getProperty("os.name").toLowerCase(Locale.ROOT))
```

Comment:
Doesn't `equalsIgnoreCase` work here?

Reply:
I doubt. We explicitly need to set the Locale.ROOT to avoid locale-sensitive conversions (see https://docs.oracle.com/en/java/javase/11/docs/api/java.base/java/lang/String.html#toLowerCase())

Also, in Platform the toLowerCase is used as well.